File: Code samples/C#/Fhi.OpenApiClient/Models/MetadataOpenResponse.cs
namespace Fhi.OpenApiClient.Models
{
    internal class MetadataOpenResponse
    {
        public string? Name { get; set; }

        public MetadataOpenParagraph[]? Paragraphs { get; set; }
    }

    internal class MetadataOpenParagraph
    {
        public string? Header { get; set; }
        public string? Content { get; set; }
    }
}

File: Code_samples/Kotlin/FhiOpenApiClient/src/main/kotlin/models/MetadataOpenResponse.kt
package models

import kotlinx.serialization.Serializable

@Serializable
internal class MetadataOpenResponse {
    var name: String? = null;
    var paragraphs: Array<MetadataOpenParagraph>? = null;

    @Serializable
    internal class MetadataOpenParagraph {
        var header: String? = null;
        var content: String? = null;
    }
}